### ISSUE TITLE
No module named 'segtools.numpy_utils'

### ISSUE THREAD

**Issue by Sumching** [mpicbg-csbd/structured_N2V#1]
Hi,  
Did you build that package segtools yourself? If I install segtools with pip, I get this error.

**Comment by colemanbroad** [comment]
Yes, we did. Both this repo and the [segtools repo](https://github.com/mpicbg-csbd/detsegtra) are unstable, and will likely require a bit of tinkering to run. However, the StructN2V method has been integrated into the [JugLab's N2V Implementation](https://csbdeep.bioimagecomputing.com/tools/n2v/) and has an [example IPython notebook](https://github.com/juglab/n2v/blob/master/examples/2D/structN2V_2D_synth_mem/train_and_predict.ipynb). I hope it works for you!
